src/OpenTelemetry.Exporter.InMemory/InMemoryExporterOptions.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="InMemoryExporterOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+
+namespace OpenTelemetry.Exporter
+{
+    public class InMemoryExporterOptions
+    {
+        public ICollection<object> Collection { get; set; }
```

Comment:
`Collection` is a little generic, maybe we should use something like `ExportedItems`?

Reply:
That's a much better name 👍